CMakeLists.txt

```diff
@@ -33,6 +33,21 @@ target_include_directories(firebase INTERFACE
 if(ANDROID)
   target_link_directories(firebase INTERFACE
     third_party/firebase-development/usr/libs/android/${CMAKE_ANDROID_ARCH_ABI})
+  if(CMAKE_ANDROID_ARCH STREQUAL arm64)
```

Comment:
Worth adding a comment here on why this manual linking is required for those that end up in this file after this PR merges?

Reply:
Sure, I can add that comment, except, its not manually linking anything - its adjusting the library search path.

Reply:
Ah sorry, I misread the description! Yeah I think a comment on _why_ we're adjusting the paths is probably helpful for those of us (me) that eventually land in this code :P

Reply:
Yeah, nothing against the comment :)